rust/operator-binary/src/controller.rs

```diff
@@ -392,6 +437,12 @@ fn build_server_rolegroup_daemonset(
                     }),
                     ..Volume::default()
                 })
+                .add_volume(Volume {
+                    name: "bundles".to_string(),
+                    empty_dir: Some(EmptyDirVolumeSource::default()),
```

Comment:
Not sure here, but do we want this to be an emtpydir or maybe rather something in memory, as the rules can contain sensitive data like user-group assignments and we'd want to lock that down as much as possible.

Reply:
Done.